bitnami/discourse/templates/deployment-sidekiq.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "discourse.fullname" . }}-sidekiq
+  labels:
+    {{- include "discourse.labels" . | nindent 4 }}
+    component: sidekiq
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "discourse.selectorLabels" . | nindent 6 }}
+      component: sidekiq
+  template:
+    metadata:
+      labels:
+        {{- include "discourse.selectorLabels" . | nindent 8 }}
+        component: sidekiq
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ include "discourse.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default "latest" }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/app-entrypoint.sh"]
+          args:
+            - nami
+            - start
+            - --foreground
+            - discourse-sidekiq
+          env:
+            - name: DISCOURSE_HOST
+              value: {{ include "discourse.fullname" . }}
+            - name: DISCOURSE_PORT
+              value: "3000"
+          envFrom:
+            - configMapRef:
+                name: {{ include "discourse.fullname" . }}-envs
+            - secretRef:
+                name: {{ include "discourse.fullname" . }}-envs
+            # {{ .Values.additionalEnvs | toYaml | indent 6 }}
```

Comment:
Cool! I was going to suggest this as well. We normally call it extraEnv, and we allow setting it as an array, as config map or as a secret. Once again I think it makes sense to have a discourse: an a sidekiq: section where you can define each of them.

See our kibana chart for an example in extraEnv, extraEnvCM and extraEnvSecret